client/dashboard/task-list/tasks/appearance.js

```diff
@@ -200,7 +209,12 @@ class Appearance extends Component {
 						<Button isPrimary onClick={ this.createHomepage }>
 							{ __( 'Create homepage', 'woocommerce-admin' ) }
 						</Button>
-						<Button onClick={ () => this.completeStep() }>
+						<Button
+							onClick={ () => {
+								recordEvent( 'tasklist_appearance_create_homepage', { skipped: true } );
```

Comment:
Should the params for this and the above `tasklist_appearance_create_homepage ` just be `skipped` or `created` to make filtering these later easier?  Or does two different parameters allow us to better filter these events?

Reply:
Can you clarify what you mean?  Do you mean something like below?

`recordEvent( 'tasklist_appearance_create_homepage', { action: 'skipped' } )`?

Looking at the Funnels/Trakcks UI, it looks like we should be able to filter by either style.

Reply:
Sure, what I meant was something to the effect of 

```recordEvent( 'tasklist_appearance_create_homepage', { action_taken: true|false } )```

`action_taken` is probably not a great name here, but my thinking was it would be easier to filter by true/false than a different property.  If that doesn't make things easier to grok on the events side, we can ignore and merge as is.

Reply:
Gotcha. I went ahead and updated these in https://github.com/woocommerce/woocommerce-admin/pull/2991/commits/1265eca53b0b8568b7f32ed29d050e8310f42a6f.